manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com//opendatahub-io/notebooks/tree/main/habana"
+    opendatahub.io/notebook-image-name: "HabanaAI"
+    opendatahub.io/notebook-image-desc: "Jupyter notebook image with a set of habana libraries that advanced AI/ML notebooks will use as a base image to provide a standard for libraries avialable in all notebooks"
+    opendatahub.io/notebook-image-order: "7"
+    opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
+  name: jupyter-habana-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  # 1.11.0 Version of the image n
+  - annotations:
+      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"},{"name":"Habana","version":"1.11.0.587"}]'
```

Comment:
Do we want to display the full version?

Reply:
Thanks for pointing that, i have change it to x.y format.